File: src/main/java/pe/com/test_qsystem/distribution_center/impl/HuanucoDistributionCenterImpl.java
package pe.com.test_qsystem.distribution_center.impl;

import pe.com.test_qsystem.distribution_center.DistributionCenter;

public class HuanucoDistributionCenterImpl implements DistributionCenter {

    @Override
    public boolean support(String center) {
        return center.equalsIgnoreCase("HUANUCO");
    }

    @Override
    public void operate(String center) {
        System.out.println("Operate in Huanuco distribution center.");
    }
}

File: src/main/kotlin/pe/com/qsystem/test_qsystem/distribution_center/impl/HuanucoDistributionCenterImpl.kt
package pe.com.qsystem.test_qsystem.distribution_center.impl

import org.slf4j.LoggerFactory
import pe.com.qsystem.test_qsystem.distribution_center.DistributionCenter

class HuanucoDistributionCenterImpl : DistributionCenter {
        
    private val log = LoggerFactory.getLogger(javaClass)

    override fun support(center: String) = center.contentEquals("HUANUCO", true)

    override fun operate(center: String) {
        log.info("Operate in Huanuco distribution center.")
    }
}